avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -586,6 +586,9 @@ def updatePositionC(cfg, particles, dem, force):
       uyNew = uyNew * uMag / (uMagNew + velMagMin)
       uzNew = uzNew * uMag / (uMagNew + velMagMin)
 
+    if uMag > 0.01:
```

Comment:
This setting needs to go to the ini file

Reply:
agreed, how should I call it? I just did not like the fact that for the mass stopping criterion you need an extra input parameter.